pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -247,15 +247,15 @@ def __init__(self, size_word, threshold, data_ts, base_cardinality=2, max_card_a
         self.cdf_mean = None
         self.cdf_std = None
 
-        # attribut après une mise à jour de l'arbre
+        # Attribute after an update of the tree
         self._new_insertion_after_preproc = False
         self._new_insertion_after_minmax_nodes = False
 
     def insert(self, new_sequence):
         """
-        La fonction insert converti en PAA puis appel la fonction insert_paa
+        The converted insert function in PAA then call the function insert_paa
```

Comment:
This `insert` function convert `new sequence` in PAA values then call the function `insert_paa`

Reply:
```suggestion
        This insert function convert new sequence in PAA values then call the function insert_paa
```